sdk/canton/daml-common-staging/daml-errors/src/main/scala/com/daml/error/SerializableErrorComponents.scala

```diff
@@ -215,7 +215,7 @@ private[error] final case class NonSecuritySensitiveErrorCodeComponents(
 private[error] object NonSecuritySensitiveErrorCodeComponents {
 
   /** The maximum size (in characters) of the self-service error description, truncated for transport as part of a Status */
-  val MaxCauseLogLength = 512
+  val MaxCauseLogLength = 1024
```

Comment:
fyi @dylant-da and @remyhaemmerle-da , this part still needs to be "reverted" via cherry-pick as the sdk/daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/upgrade/UpgradesSpecBase.scala - based tests still assert long error messages from canton

I took a stab at adapting the test, but this ended up messing up the test quite badly, so I did not pursue this route: 
[daml-lf-backup-UpgradesSpecBase.scala.txt](https://github.com/user-attachments/files/17476714/daml-lf-backup-UpgradesSpecBase.scala.txt)